internal/controllers/module_nmc_reconciler.go

```diff
@@ -402,6 +408,37 @@ func (mnrh *moduleNMCReconcilerHelper) removeModuleFromNMC(ctx context.Context,
 	return nil
 }
 
+func (mnrh *moduleNMCReconcilerHelper) moduleUpdateWorkerPodsStatus(ctx context.Context, mod *kmmv1beta1.Module, targetedNodes []v1.Node) error {
+	logger := log.FromContext(ctx)
+	// get nmcs with configured
+	nmcs, err := mnrh.getNMCsForModule(ctx, mod)
+	if err != nil {
+		return fmt.Errorf("failed to get configured NMCs for module %s/%s: %v", mod.Namespace, mod.Name, err)
+	}
+
+	numAvailable := 0
+	for _, nmc := range nmcs {
+		modSpec, _ := mnrh.nmcHelper.GetModuleSpecEntry(&nmc, mod.Namespace, mod.Name)
+		if modSpec == nil {
+			logger.Info(utils.WarnString(
+				fmt.Sprintf("module %s/%s spec is missing in NMC %s although config label is present", mod.Namespace, mod.Name, nmc.Name)))
```

Comment:
I think we should use log fields instead of `fmt.Sprintf`.

Reply:
This is a part of Warning message, that is how we use it throughout our code for better visibility